<ui version="4.0" >
 <class>knoGameDialog</class>
 <widget class="QDialog" name="knoGameDialog" >
  <property name="windowModality" >
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>740</width>
    <height>570</height>
   </rect>
  </property>
  <property name="minimumSize" >
   <size>
    <width>740</width>
    <height>570</height>
   </size>
  </property>
  <property name="maximumSize" >
   <size>
    <width>740</width>
    <height>570</height>
   </size>
  </property>
  <property name="windowTitle" >
   <string>KNO - Game</string>
  </property>
  <property name="modal" >
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="layoutWidget" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>0</y>
     <width>722</width>
     <height>554</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" >
    <item>
     <layout class="QVBoxLayout" >
      <item>
       <widget class="poleWidget" name="poleView" >
        <property name="minimumSize" >
         <size>
          <width>440</width>
          <height>440</height>
         </size>
        </property>
        <property name="maximumSize" >
         <size>
          <width>1000</width>
          <height>1000</height>
         </size>
        </property>
        <property name="palette" >
         <palette>
          <active>
           <colorrole role="WindowText" >
            <brush brushstyle="SolidPattern" >
             <color alpha="255" >
              <red>0</red>
              <green>0</green>
              <blue>0</blue>
             </color>
            </brush>
           </colorrole>
           <colorrole role="HighlightedText" >
            <brush brushstyle="SolidPattern" >
             <color alpha="255" >
              <red>255</red>
              <green>255</green>
              <blue>255</blue>
             </color>
            </brush>
           </colorrole>
          </active>
          <inactive>
           <colorrole role="WindowText" >
            <brush brushstyle="SolidPattern" >
             <color alpha="255" >
              <red>0</red>
              <green>0</green>
              <blue>0</blue>
             </color>
            </brush>
           </colorrole>
           <colorrole role="HighlightedText" >
            <brush brushstyle="SolidPattern" >
             <color alpha="255" >
              <red>255</red>
              <green>255</green>
              <blue>255</blue>
             </color>
            </brush>
           </colorrole>
          </inactive>
          <disabled>
           <colorrole role="WindowText" >
            <brush brushstyle="SolidPattern" >
             <color alpha="255" >
              <red>106</red>
              <green>104</green>
              <blue>100</blue>
             </color>
            </brush>
           </colorrole>
           <colorrole role="HighlightedText" >
            <brush brushstyle="SolidPattern" >
             <color alpha="255" >
              <red>255</red>
              <green>255</green>
              <blue>255</blue>
             </color>
            </brush>
           </colorrole>
          </disabled>
         </palette>
        </property>
        <property name="toolTip" >
         <string/>
        </property>
        <property name="whatsThis" >
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" >
        <item>
         <layout class="QVBoxLayout" >
          <item>
           <layout class="QHBoxLayout" >
            <item>
             <layout class="QHBoxLayout" >
              <item>
               <widget class="QLabel" name="label" >
                <property name="text" >
                 <string>You are</string>
                </property>
                <property name="buddy" >
                 <cstring>youAreEdit</cstring>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="youAreEdit" >
                <property name="enabled" >
                 <bool>true</bool>
                </property>
                <property name="inputMask" >
                 <string/>
                </property>
                <property name="text" >
                 <string/>
                </property>
                <property name="cursorPosition" >
                 <number>0</number>
                </property>
                <property name="readOnly" >
                 <bool>true</bool>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" >
              <item>
               <widget class="QLabel" name="label_2" >
                <property name="text" >
                 <string>Next turn</string>
                </property>
                <property name="buddy" >
                 <cstring>nextTurnEdit</cstring>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="nextTurnEdit" >
                <property name="inputMask" >
                 <string/>
                </property>
                <property name="text" >
                 <string/>
                </property>
                <property name="readOnly" >
                 <bool>true</bool>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
          <item>
           <widget class="Line" name="line" >
            <property name="orientation" >
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" >
            <item>
             <layout class="QVBoxLayout" >
              <item>
               <spacer>
                <property name="orientation" >
                 <enum>Qt::Vertical</enum>
                </property>
                <property name="sizeHint" >
                 <size>
                  <width>20</width>
                  <height>40</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item>
               <widget class="QLabel" name="label_5" >
                <property name="text" >
                 <string>Result:</string>
                </property>
               </widget>
              </item>
              <item>
               <spacer>
                <property name="orientation" >
                 <enum>Qt::Vertical</enum>
                </property>
                <property name="sizeHint" >
                 <size>
                  <width>20</width>
                  <height>40</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" >
              <item>
               <layout class="QVBoxLayout" >
                <item>
                 <widget class="QLabel" name="label_3" >
                  <property name="text" >
                   <string>You</string>
                  </property>
                  <property name="buddy" >
                   <cstring>youEdit</cstring>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLabel" name="label_4" >
                  <property name="text" >
                   <string>Enemy</string>
                  </property>
                  <property name="buddy" >
                   <cstring>enemyEdit</cstring>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QVBoxLayout" >
                <item>
                 <widget class="QLineEdit" name="youEdit" >
                  <property name="inputMask" >
                   <string/>
                  </property>
                  <property name="text" >
                   <string>0</string>
                  </property>
                  <property name="readOnly" >
                   <bool>true</bool>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLineEdit" name="enemyEdit" >
                  <property name="inputMask" >
                   <string/>
                  </property>
                  <property name="text" >
                   <string>0</string>
                  </property>
                  <property name="readOnly" >
                   <bool>true</bool>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <widget class="Line" name="line_3" >
          <property name="orientation" >
           <enum>Qt::Vertical</enum>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QVBoxLayout" >
          <item>
           <widget class="QPushButton" name="newGameButton" >
            <property name="enabled" >
             <bool>false</bool>
            </property>
            <property name="focusPolicy" >
             <enum>Qt::NoFocus</enum>
            </property>
            <property name="text" >
             <string>New game (only for servers)</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="Line" name="line_2" >
            <property name="orientation" >
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="helpButton" >
            <property name="focusPolicy" >
             <enum>Qt::NoFocus</enum>
            </property>
            <property name="text" >
             <string>Help</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="aboutGameButton" >
            <property name="focusPolicy" >
             <enum>Qt::NoFocus</enum>
            </property>
            <property name="text" >
             <string>About KNO</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" >
      <item>
       <widget class="QTextEdit" name="chatEdit" >
        <property name="enabled" >
         <bool>true</bool>
        </property>
        <property name="minimumSize" >
         <size>
          <width>270</width>
          <height>500</height>
         </size>
        </property>
        <property name="maximumSize" >
         <size>
          <width>270</width>
          <height>500</height>
         </size>
        </property>
        <property name="palette" >
         <palette>
          <active>
           <colorrole role="HighlightedText" >
            <brush brushstyle="SolidPattern" >
             <color alpha="255" >
              <red>255</red>
              <green>255</green>
              <blue>255</blue>
             </color>
            </brush>
           </colorrole>
          </active>
          <inactive>
           <colorrole role="HighlightedText" >
            <brush brushstyle="SolidPattern" >
             <color alpha="255" >
              <red>255</red>
              <green>255</green>
              <blue>255</blue>
             </color>
            </brush>
           </colorrole>
          </inactive>
          <disabled>
           <colorrole role="HighlightedText" >
            <brush brushstyle="SolidPattern" >
             <color alpha="255" >
              <red>255</red>
              <green>255</green>
              <blue>255</blue>
             </color>
            </brush>
           </colorrole>
          </disabled>
         </palette>
        </property>
        <property name="font" >
         <font>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="readOnly" >
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="sayEdit" />
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>poleWidget</class>
   <extends>QWidget</extends>
   <header>polewidget.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>youAreEdit</tabstop>
  <tabstop>nextTurnEdit</tabstop>
  <tabstop>newGameButton</tabstop>
  <tabstop>youEdit</tabstop>
  <tabstop>enemyEdit</tabstop>
  <tabstop>helpButton</tabstop>
  <tabstop>aboutGameButton</tabstop>
  <tabstop>chatEdit</tabstop>
  <tabstop>sayEdit</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
